spec/models/event_spec.rb

```diff
@@ -84,6 +85,15 @@
         expect(described_class.count - @pre_count).to eq(1)
       end
 
+      context 'with an occurred_at attribute' do
+        let(:occured_at) { 'occurred_at' }
+
+        it 'translates to occured_at' do
+          expect(described_class.count - @pre_count).to eq(1)
+          expect(described_class.last.occured_at).to eq(Time.zone.parse('2012-03-11 10:22:42'))
+        end
+      end
```

Comment:
I think this might be a bit confusing out of context 😆  maybe add a brief comment like `hint: look at the spelling`

Reply:
Haha, good point